tutorials/02_installation.md

```diff
@@ -44,15 +44,17 @@ Be sure to replace `<#>` with a number value, such as `1` or `2`, depending on w
 1. Install dependencies
   ```
   sudo apt-add-repository -s "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -c -s) main"
-  sudo apt-get build-dep -y gz-physics<#>-dev
+  sudo apt-get build-dep -y libgz-physics<#>-dev
   ```
-  Be sure to replace `<#>` with a number value, such as `1` or `2`, depending on which version you need.
+  Be sure to replace `<#>` with a number value, such as `5` or `6`, depending on which version you need.
+  Older versions can be installed by using the `libignition-physics<#>-dev` packages.
```

Comment:
can we specify which are the older versions?

Reply:
Yes.

88b8ada4507f0cc03a15dc3abf599589f9f64d25